EAD-Assignment-4/WebContent/WebContent/Scripts/scriptforlogin.js

```diff
@@ -0,0 +1,44 @@
+function checkValidationLogin(){
+	
+	var email = document.getElementById("email").value;
+	var pwd = document.getElementById("pwd");
+	if(isEmailValidLogin() && isPasswordValidLogin())
+	{
+		var emailstr="arjita101@gmail.com";
+		var pwdstr="Arjita@1234";
+		if(email == emailstr && pwd == pwdstr)
+		{
+			document.getElementById("form").submit();
+			return true;
+		}
+		else{
+			alert("Wrong email or password.")
+			return false;
+		}
+	}
+	else{
+		alert("Invalid email or password")
+		return false;
+	}
+}
+function isEmailValidLogin() {

```

Comment:
Can use the same functions defined in the script.js file, instead of rewriting the functions